src/pages/mesh/release/index.md

```diff
@@ -18,6 +18,16 @@ import UpdateNotice from '/src/_includes/update-notice.md'
 
 The following sections list updates to API Mesh for Adobe Developer App Builder. Refer to the [Upgrade version](upgrade.md) for more information on upgrading.
 
+## November 04, 2024
+
+This release contains the following changes to API Mesh:
+
+### Enhancements
+
+- [`beforeAll` hooks](../advanced/hooks.md) now work with [edge meshes](../basic/create-mesh.md#access-your-mesh-urls).
```

Comment:
Line 17 of the `hooks.md` file says `beforeAll` hooks were the only supported type. Based on the deleted text within that file, it seems like this line should start with "All hooks".

Reply:
This line just re-emphasizes that `beforeAll` hooks are the only available hooks.